src/main/kotlin/com/atlassian/performance/tools/awsinfrastructure/api/aws/AwsCli.kt

```diff
@@ -12,7 +12,14 @@ import java.util.concurrent.ConcurrentHashMap
  *
  * @since 2.15.0
  */
-class AwsCli {
+class AwsCli (val cliVersion: String = "2.9.12") {
```

Comment:
I've changed the default version to 2.9.12.  The bulk copying in `SharedHomeFormula.kt` wasn't reliable with the V1 CLI because it only supports legacy retry mode not [standard retry mode](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-retries.html#cli-usage-retries-modes-standard.title).  The new code can install a V1 or V2 cli and I've tested both, though it uses 2.9.12 by default everywhere now which I'm not sure if you'll be ok with or not.